### ISSUE TITLE
[Development] Add validation to CVE field values for RedHat feeds

### ISSUE THREAD

**Issue by jmv74211** [wazuh/wazuh#5194]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.13.0-31302 | Vulnerability detector | Manage | Packages | CentOS 7 |

Hello team,

The CICD team is working on the integration tests for the vulnerability detector, and we would like to comment on the following case.

It seems that any string type value is accepted in the `CVE` field of the Redhat feed, that is, there is not a value validation for that field. For example, if you have the following value

```
"CVE": ""
```

You can see that the vulnerability has been inserted correctly with the following content:

```
|RHEL8|1|wazuhintegrationpackage-0|less than|1.0.el8_1.2|0|0
```

The same occurs for another series of values such as the following

```
 ["", "dummy value", "ñ", "テスト", "ИСПЫТАНИЕ", "测试", "اختبار"]
```
Even an emoticon

![image](https://user-images.githubusercontent.com/23462183/84148285-0e7e6180-aa5f-11ea-8273-48236f0bb612.png)

In conclusion, any field that is a string.

It is proposed to carry out some kind of validation of this value since it is one of the main fields when parsing the feed and inserting it in the corresponding table/s of the database.

Best regards.
